src/main/webapp/mapScript.js

```diff
@@ -45,14 +50,15 @@ function initMap() {
           map: map,
         });
 
-        let index = markersArray.length,
+        let index = globalIndex,
```

Comment:
I..... don't actually understand what are the pros of global variable, can you please explain?

Reply:
I can't say that it is such a good thing to add a global variable, but the previous approach was not functionally correct, because the array length was changing (for example when deleting markers), and because of that the correspondence between the array index and the place id broke down.

Reply:
ooh, got it!